src/packages/sdk/src/__tests__/getSchema.test.ts

```diff
@@ -113,8 +113,8 @@ it('reads schema path from the nearest package.json and throws if path does not
 
   expect(res).toMatchInlineSnapshot(`
     Object {
-      "async": [Error: Provided schema path at ./wrong-path from ./package.json doesn't exist.],
-      "sync": [Error: Provided schema path at ./wrong-path from ./package.json doesn't exist.],
+      "async": [Error: Provided schema path at \`wrong-path\` from \`package.json\` doesn't exist.],
+      "sync": [Error: Provided schema path at \`wrong-path\` from \`package.json\` doesn't exist.],
```

Comment:
Is that a bit confusing maybe?
I would get rid of the `at` I think because it's suggesting to look at a path but here we mean the value from the package.json like
```
{
  "name": "pkg-json-valid-path",
  "prisma": {
    "schema": "./wrong-path"
  }
}
```

Reply:
Meaning something like `Provided schema path \`wrong-path\` from \`package.json\` doesn't exist` ?

Reply:
Yes, does that make sense to you too?